src/mscorlib/model.xml

```diff
@@ -65,10 +65,33 @@
       <Member Status="ApiRoot" Name="Invoke(T1,T2,T3,T4,T5,T6,T7,T8)" />
     </Type>
     <Type Name="System.Activator">
+      <Member Name="CreateComInstanceFrom(System.String,System.String)" />
+      <Member Name="CreateComInstanceFrom(System.String,System.String,System.Byte[],System.Configuration.Assemblies.AssemblyHashAlgorithm)" />
+      <Member Name="CreateInstance(System.AppDomain,System.String,System.String)" />
+      <Member Name="CreateInstance(System.AppDomain,System.String,System.String,System.Boolean,System.Reflection.BindingFlags,System.Reflection.Binder,System.Object[],System.Globalization.CultureInfo,System.Object[])" />
+      <Member Name="CreateInstance(System.AppDomain,System.String,System.String,System.Boolean,System.Reflection.BindingFlags,System.Reflection.Binder,System.Object[],System.Globalization.CultureInfo,System.Object[],System.Security.Policy.Evidence)" />
```

Comment:
This looks like it uses System.Security.Policy.Evidence which isn't exposed, and we currently don't plan to expose. Can you please exclude the overloads that take that as a parameter?